test/simple_test_ODWB.jl

```diff
@@ -0,0 +1,151 @@
+############## A optimal design ######################################################################
+# Problem described here: https://link.springer.com/article/10.1007/s11222-014-9476-y
+# "A first-order algorithm for the A-optimal experimental design Problem: a mathematical programming approach"
+
+# min 1/(trace(∑x_i v_iv_i^T))
+# s.t. \sum x_i = s
+#       lb ≤ x ≤ ub
+#       x ∈ Z^m
+
+# v_i ∈ R^n
+# n - number of parameters
+# m - number of possible experiments
+# A = [v_1^T,.., v_m^T], so the rows of A correspond to the different experiments
+
+
+################################ D optimal design ########################################################################
+# Problem described here: https://arxiv.org/pdf/2302.07386.pdf
+# "Branch-and-Bound for D-Optimality with fast local search and bound tightening"
+
+# min log(1/(det(∑x_i v_iv_i^T)))
+# s.t. \sum x_i = s
+#       lb ≤ x ≤ ub
+#       x ∈ Z^m
+
+# v_i ∈ R^n
+# n - number of parameters
+# m - number of possible experiments
+# A = [v_1^T,.., v_m^T], so the rows of A correspond to the different experiments
+
+################################ D-fusion design ########################################################################
+# Problem described here: https://arxiv.org/pdf/2302.07386.pdf
+# "Branch-and-Bound for D-Optimality with fast local search and bound tightening"
+
+# min log(1/(det(∑x_i v_iv_i^T)))
+# s.t. \sum x_i = s
+#       lb ≤ x ≤ ub
+#       x ∈ Z^m
+
+# v_i ∈ R^n
+# n - number of parameters
+# m - number of possible experiments
+# A = [v_1^T,.., v_m^T], so the rows of A correspond to the different experiments
+
+using Boscia
+using FrankWolfe
+using Bonobo
+using Random
+using SCIP
+using JuMP
+using Hypatia
+import Hypatia.Cones: vec_length, vec_copyto!, svec_length, svec_side
+import Hypatia.Cones: smat_to_svec!, svec_to_smat!
+const Cones = Hypatia.Cones
+using Pajarito
+using PajaritoExtras # https://github.com/chriscoey/PajaritoExtras.jl

```

Comment:
You don't need these dependencies for testing. They are used for the experiments only.